wrappers/python/simtk/openmm/app/topology.py

```diff
@@ -155,7 +182,7 @@ def addAtom(self, name, element, residue, id=None):
         residue._atoms.append(atom)
         return atom
 
-    def addBond(self, atom1, atom2):
+    def addBond(self, atom1, atom2, type=None, order=None):
```

Comment:
Should *both* `type` and `order` be acceptable?
What if `type` and `order` are both specified, but conflict with each other? Like 
```python
topology.addBond(atom1, atom2, type=Double, order=1)
```
?